template_example/dev/ansible/03_healthcheck_test.yml

```diff
@@ -0,0 +1,34 @@
+- name: set variable
+  set_fact:
+    consul_token: "master"
+  tags: test
+
+- name: countdash healthcheck test without consul token (when consul acl default policy is allow)
+  uri:
+    url: http://localhost:8500/v1/health/checks/count-dashboard?filter={{ '(Status=="passing")' | urlencode }}
+    method: GET
+    return_content: yes
+    status_code: 200
+    body_format: json
+  when: not lookup('env', 'consul_acl') | bool
+  register: result_countdashboard
+  retries: 15
+  delay: 15
+  until: result_countdashboard.json | length == 1
+  tags: test
+
+- name: countdash healthcheck test with consul token (when consul acl default policy is deny)
+  uri:
+    url: http://localhost:8500/v1/health/checks/count-dashboard?filter={{ '(Status=="passing")' | urlencode }}
+    method: GET
+    headers:
+      X-Consul-Token: "{{ consul_token }}"
+    return_content: yes
+    status_code: 200
+    body_format: json
+  when: lookup('env', 'consul_acl') | bool and lookup('env', 'consul_acl_default_policy') == 'deny'
+  register: result_countdashboard
+  retries: 15
+  delay: 15
+  until: result_countdashboard.json | length == 1
+  tags: test
```

Comment:
I would recommend having an additional check for existing **intentions** between `count-dashboard` ->`count-api` and

Reply:
Setting the variable explicitly at the beginning of the file, has been a conscious choice as the consul_master_token is not available to use as we are not using the 0.6.0 version of the box. This variable is available only in 0.6.0 version of the box.

Reply:
Also, by **check for existing intentions ** do you mean that we should create intentions like we have done it in other terraform modules?

Reply:
We discussed it previously, that services `count-dashboard` and `count-api` (when consul_acl_default_policy=deny) do not communicate with each other (because intentions are not existed) and existing health checks do not show the problem.

Reply:
Yeah, i do remember. Somehow, i am unable to access the count dashboard even after creating the proxy so unable to proceed. This means it's going to take some time.

Reply:
@zhenik: I have added intentions and a test to verify the connection.